http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LeoProfile.scala

```diff
@@ -313,6 +313,20 @@ private[leonardo] object LeoProfile extends MySQLProfile {
 
     implicit val instantSetParameter: SetParameter[Instant] =
       SetParameter.SetTimestamp.contramap(instant => java.sql.Timestamp.from(instant))
+
+    implicit val appUpgradeTableIdColumnType: BaseColumnType[UpdateAppTableId] =
+      MappedColumnType.base[UpdateAppTableId, Long](_.value, UpdateAppTableId.apply)
+
+    implicit val appUpgradeJobIdColumnType: BaseColumnType[UpdateAppJobId] =
```

Comment:
Upgrade? should this be update

Reply:
We should pick one and stick with it, I do not really care which one but it looks like `Update` was the one previously used?